runtime/compiler/runtime/JITClientSession.cpp

```diff
@@ -680,8 +680,10 @@ ClientSessionHT::allocate()
    }
 
 ClientSessionHT::ClientSessionHT() : _clientSessionMap(decltype(_clientSessionMap)::allocator_type(TR::Compiler->persistentAllocator())),
-                                     TIME_BETWEEN_PURGES(1000*60*30), // JITServer TODO: this must come from options
-                                     OLD_AGE(1000*60*1000) // 1000 minutes
+                                     TIME_BETWEEN_PURGES(TR::Options::_timeBetweenPurges), // JITServer TODO: this must come from options
+                                     OLD_AGE(TR::Options::_oldAge), // 1000 minutes
```

Comment:
Let's put `OLD_AGE` and `OLD_AGE_UNDER_LOW_MEMORY` one after another because they are related